src/Access.php

```diff
@@ -524,6 +524,25 @@ public function send_welcome_email( $user_id, $password = '' ) {
 			$email_subject = nl2br( $this->settings->ssc_get_option( 'ssc_email_subject' ) );
 			$pages         = '';
 			$user          = get_user_by( 'ID', $user_id );
+
+			// Apply the new password logic only for the new users to not reset password for existing users.
+			if ( get_user_meta( $user_id, 'user_registered', true ) > '2022-08-01 00:00:00' ) {
```

Comment:
Řešení se mi líbí, ale nemůžeme to vázat na datum update. Nejsme schopni určit, kdy se dostane do konkrétní instance WP. 

Nápad (ale nevím, nakolik je validní) tuhle logiku ještě trochu otočit: Při vytváření uživatelského účtu si do meta poznačit, že je to uživatel nově vytvořený SS pluginem a následně pak po odeslání e-mailu s heslem tento příznak smazat. 

Tím bude zajištěno, že starší uživatelé nebudou dotčeni a bude to týkat jen těch, o kterých víme s jistotou, že je založil SS plugin.